packages/google_maps_flutter/test/fake_maps_controllers.dart

```diff
@@ -96,6 +103,9 @@ class FakePlatformGoogleMap {
       case 'circles#update':
         updateCircles(call.arguments);
         return Future<void>.sync(() {});
+      case 'heatmaps#update':
+        updateHeatmaps(call.arguments);
+        return Future<void>.sync(() {});
```

Comment:
This should be converted to a `Stream<HeatmapUpdateEvent>` in the `platform_interface` (all the code of onMethodCall has migrated to that package as well).